src/components/Filter/LessonFilter.js

```diff
@@ -36,15 +37,17 @@ const LessonFilter = ({filterGroupKeys, isStudentMode, t}) => {
       <Col xsHidden>
         <Panel {...{header, bsStyle}}>
           <ListGroup fill>
-            {filterGroups}
+            <RadioButtons/>
```

Comment:
Jeg synes sjekken for om radiobuttons skal vises eller ikke (mtp language) burde også være her.